driver-scala/src/main/scala/org/mongodb/scala/MongoCredential.scala

```diff
@@ -29,23 +29,23 @@ object MongoCredential {
   /**
    * The GSSAPI mechanism.  See the <a href="http://tools.ietf.org/html/rfc4752">RFC</a>.
    *
-    @see [[http://docs.mongodb.org/manual/core/authentication/#kerberos-authentication GSSAPI]]
+    @see [[https://www.mongodb.com/docs/manual/core/authentication/#kerberos-authentication GSSAPI]]
    * @since 4.0
    */
   val GSSAPI_MECHANISM: String = JMongoCredential.GSSAPI_MECHANISM
 
   /**
    * The PLAIN mechanism.  See the <a href="http://www.ietf.org/rfc/rfc4616.txt">RFC</a>.
    *
-    @see [[http://docs.mongodb.org/manual/core/authentication/#ldap-proxy-authority-authentication PLAIN]]
+    @see [[https://www.mongodb.com/docs/manual/core/authentication/#ldap-proxy-authority-authentication PLAIN]]
```

Comment:
Looks like this refers to an old target.
New link suggestion: https://www.mongodb.com/docs/manual/core/security-ldap/